README.md

```diff
@@ -183,6 +189,31 @@ In any case the exact creation time of any `v1` UUID will be contained within th
 can be a privacy or data protection concern for many use cases (e.g. leaking the creation timestamp
 of a user account) so it's yet another reason to be very careful when choosing to use `v1` UUIDs.
 
+### How do other languages/libraries deal with UUIDs?
+
+Other languages/libraries seem to do be using the term `random` to describe `v4` UUIDs as well
+([go](https://godoc.org/github.com/google/uuid#NewRandom),
+[Java](<https://docs.oracle.com/javase/10/docs/api/java/util/UUID.html#randomUUID()>),
+[C++ Boost](https://www.boost.org/doc/libs/1_71_0/boost/uuid/random_generator.hpp)).
+
+Apart from that, UUID adoption in other languages/libraries seems to be rather inconsistent:
+
+- [Java](<https://docs.oracle.com/javase/10/docs/api/java/util/UUID.html#randomUUID()>) provides
+  methods for generating `v3` and `v4` UUIDs but not `v1` or `v5` (pretty weird why there's `v3`
```

Comment:
I don't love the phrasing "pretty weird", perhaps:

> it would be interesting to investigate further, as to why these algorithms were chosen, given it goes against the recommendations in the RFC.

Reply:
💯agreed, that sloppy wording was a copy&paste leftover from my comment in #31 

While I still do find it a weird choice (😈) I agree that this kind of negative judgement should not appear in the README, thanks for pointing it out!

Reply:
💯agreed, that sloppy wording was a copy&paste leftover from my comment in #31 

While I still do find it a weird choice (😈) I agree that this kind of negative judgement should not appear in the README, thanks for pointing it out!